src/components/widgets/DefaultWidget.tsx

```diff
@@ -16,11 +16,11 @@ type Props = {
 export const DefaultWidget = ({ Icon, count, onPress, text }: Props) => {
   return (
     <TouchableOpacity onPress={onPress}>
-      <WrapperVertical style={styles.container}>
+      <WrapperVertical style={styles().container}>
         <WrapperRow center>
-          <Icon style={styles.icon} />
+          <Icon style={styles(!!count?.toString()).icon} />
           <BoldText primary big>
-            {count ?? ' '}
+            {count}
```

Comment:
can you think of a case where `count` is something that results in rendering a `null` or `undefined` or something on the screen? do we need to be more safe here somehow?

Reply:
if the `count` value is `null` or `undefined`, it will not be displayed. i don't think the application will crash because of this.

but I can still add such a control.
<img width="395" alt="image" src="https://user-images.githubusercontent.com/11755668/188907074-66d60807-490b-447e-8ff9-7a9b61cdef1a.png">

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/621/commits/095c971020462743627d19c5c2fd946ea322aa22